.github/CONTRIBUTING.md

```diff
@@ -10,10 +10,12 @@
  - [Developer Commits Levels](#pr_forks)
  - [Submission Guidelines](#submit)
 
-## AngularJS Material is in LTS mode
+## AngularJS Material has reached End-of-Life
 
-We are no longer accepting changes that are not critical bug fixes into this project.
-See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.
+We are no longer accepting changes into this project as it has reached the end of the
+[Long Term Support](https://docs.angularjs.org/misc/version-support-status) period.
+
+Read the [end-of-life announcement](https://goo.gle/angularjs-end-of-life) for more details.
```

Comment:
Could we match the wording from https://github.com/angular/angular.js/pull/17177? 

Similar to that PR, I'd like to link to the actively supported thing so avoid any confusing (people are _hair-triggered_ on misinterpreting this as Angular)

(here and in other places)